aten/src/ATen/cuda/CUDASolver.cpp

```diff
@@ -335,6 +335,110 @@ void gesvdjBatched<c10::complex<double>>(
     lwork, info, params, batchSize));
 }
 
+template <>
+void orgqr_buffersize<float>(
```

Comment:
I saw the code for inverse and svd. I think these functions should be just templated wrappers of the underlying library and nothing more exactly mimicking the number of arguments and their order and not hide anything. So that everything is visible what's going from the `apply_*` function.
The output of `*_bufferSize` functions remains the same for the same input sizes, so there is no point in calling it inside the inner batching loop.